Lib/test/test_gettext.py

```diff
@@ -717,6 +717,84 @@ def test_expand_lang(self):
                                          return_value=locale):
                     self.assertEqual(gettext._expand_lang(locale), expanded)
 
+# helper for FindTestCase
+def _clearvars(env):
+    for key in ('LANGUAGE', 'LC_ALL', 'LC_MESSAGES', 'LANG'):
+        env.unset(key)
+
+class FindTestCase(unittest.TestCase):
+    def test_find_with_env_vars(self):
+        # test that find correctly finds the environment variable LANGUAGE
+        # when languages are not supplied
+        with os_helper.EnvironmentVarGuard() as env, os_helper.temp_cwd() as tempdir:
+            env.set('LANGUAGE', 'ga_IE')
+
+            locale_dir = os.path.join(tempdir, "locale")
+            mofile_dir = os.path.join(locale_dir, "ga_IE", "LC_MESSAGES")
+            os.makedirs(mofile_dir)
+            mo_file = os.path.join(mofile_dir, "mofile.mo")
+            with open(mo_file, "wb") as f:
+                f.write(b"\xDE\x12\x04\x95")
```

Comment:
Could you extract this to a helper method?
It's repeated 3 times, and the 4 bytes want a name/comment.